package.json

```diff
@@ -41,13 +46,16 @@
     "main": "node ./lib/rtlcss.js",
     "lint": "standard",
     "lint:fix": "npm run lint -- --fix",
-    "mocha": "mocha -R spec",
-    "mocha:special": "npm run mocha -- --fgrep \"# Special\"",
+    "mocha": "mocha",
+    "mocha:special": "mocha --fgrep \"# Special\"",
     "test": "npm run lint && npm run main && npm run mocha"
   },
-  "main": "./lib/rtlcss.js",
-  "files": [
-    "bin/*.js",
-    "lib/*.js"
-  ]
+  "mocha": {
+    "check-leaks": true,
+    "reporter": "dot",
```

Comment:
I went with `dot` but let me know if you still prefer the default spec reporter and I'll revert this change.